app/src/debug/java/io/reark/rxgithubapp/utils/StethoInstrumentation.java

```diff
@@ -73,18 +70,8 @@ void initStetho() {
     @NonNull
     public OkHttpClient decorateNetwork(@NonNull final OkHttpClient httpClient) {
         Preconditions.checkNotNull(httpClient, "Http Client cannot be null.");
-
-        addInterceptor(httpClient, interceptor);
-
-        return httpClient;
+        return httpClient.newBuilder().addNetworkInterceptor(new StethoInterceptor()).build();
```

Comment:
I was referring to the way how the StethoInterceptor is provided. The question was why won't you use dependency injection for the StethoInterceptor instead of creating it for decoration. We should use inversion of control in most case, unless there is a strong reason against it. Is there one here?